SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1283,13 +1286,18 @@ void USpatialNetDriver::ProcessOwnershipChanges()
 	{
 		if (USpatialActorChannel* Channel = GetActorChannelByEntityId(EntityId))
 		{
+			check(IsValid(Channel->Actor));
+
 			if (bShouldWriteLoadBalancingData)
 			{
-				check(IsValid(Channel->Actor));
 				const SpatialGDK::ActorSetMember ActorSetData = SpatialGDK::GetActorSetData(*PackageMap, *Channel->Actor);
 				Connection->GetCoordinator().SendComponentUpdate(EntityId, ActorSetData.CreateComponentUpdate(), {});
 			}
 
+			Connection->GetCoordinator().SendComponentUpdate(
```

Comment:
ServerProcessOwnershipChange does additional checks on this actor, and doesn't process the update if it's not correct - seems like we should move this component update there